### PR TITLE
sample code for article "Java Optional As Return Type"

### PR DESCRIPTION
**Pull request by liesheng**

### CODE REVIEW

File: core-java-modules/core-java-8-2/pom.xml [hunk 1]
```diff
@@ -30,7 +30,21 @@
             <artifactId>icu4j</artifactId>
             <version>${icu.version}</version>
         </dependency>
-
+		<dependency>
+    		<groupId>org.hibernate</groupId>
```
Comment (rpvilao) [inline core-java-modules/core-java-8-2/pom.xml]: indent

File: core-java-modules/core-java-8-2/pom.xml [hunk 2]
```diff
@@ -30,7 +30,21 @@
...
+    		<artifactId>hibernate-core</artifactId>
+    		<version>5.4.0.Final</version>
+		</dependency>
+		<dependency>
+    		<groupId>com.h2database</groupId>
+    		<artifactId>h2</artifactId>
+    		<version>1.4.197</version>
```
Comment (rpvilao) [inline core-java-modules/core-java-8-2/pom.xml]: use properties for the versioning

File: core-java-modules/core-java-8-2/pom.xml (hunk 1 above)
Comment (liesheng) [reply core-java-modules/core-java-8-2/pom.xml]: done.

File: core-java-modules/core-java-8-2/pom.xml [hunk 3]
```diff
@@ -1,50 +1,67 @@
...
+	<packaging>jar</packaging>
 
 
-    <parent>
-        <groupId>com.baeldung</groupId>
-        <artifactId>parent-java</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
-        <relativePath>../../parent-java</relativePath>
-    </parent>
+	<parent>
```
Comment (rpvilao) [inline core-java-modules/core-java-8-2/pom.xml]: 4 spaces indentation

Comment (liesheng) [reply core-java-modules/core-java-8-2/pom.xml]: done, replaced tabs with 4-spaces

**Pull request merged**